Not really sure why this test is failing:
```
12:57:34 3: [==========] Running 3 tests from 1 test suite.
12:57:34 3: [----------] Global test environment set-up.
12:57:34 3: [----------] 3 tests from PlayEndToEndTestFixture
12:57:34 3: [ RUN      ] PlayEndToEndTestFixture.play_all
12:57:34 3: [ERROR] [1610492254.934685691] [rosbag2_cpp]: Requested converter for format 'cdr' does not exist
12:57:34 3: [ERROR] [1610492254.935950959] [rosbag2_transport]: Failed to play: Could not find converter for format cdr
12:57:34 3: storage_options uri /home/jenkins-agent/workspace/packaging_linux/ws/src/ros2/rosbag2_bag_v2/rosbag2_bag_v2_plugins/resources/test_bag_end_to_end.bag
12:57:34 3: [       OK ] PlayEndToEndTestFixture.play_all (79 ms)
12:57:34 3: [ RUN      ] PlayEndToEndTestFixture.play_end_to_end_test_does_not_try_to_publish_ros1_topics
12:57:35 3: [ERROR] [1610492255.305204726] [rosbag2_cpp]: Requested converter for format 'cdr' does not exist
12:57:35 3: [ERROR] [1610492255.305302471] [rosbag2_transport]: Failed to play: Could not find converter for format cdr
12:58:34  3/12 Test  #3: test_rosbag2_play_rosbag_v2_end_to_end ...***Timeout  60.03 sec
```

is there a test-dependency missing?

--- a/rosbag2_bag_v2_plugins/test/rosbag2_bag_v2_plugins/test_rosbag2_play_rosbag_v2_end_to_end.cpp
+++ b/rosbag2_bag_v2_plugins/test/rosbag2_bag_v2_plugins/test_rosbag2_play_rosbag_v2_end_to_end.cpp
@@ -69,6 +69,7 @@ TEST_F(PlayEndToEndTestFixture, play_all) {
   printf("storage_options uri %s\n", storage_options.uri.c_str());
   storage_options.storage_id = "rosbag_v2";
   rosbag2_transport::PlayOptions play_options;
+  play_options.read_ahead_queue_size = 1000;
   rosbag2_transport.play(storage_options, play_options);
 
   SUCCEED();

